cve_bin_tool/cvedb.py

```diff
@@ -253,7 +253,7 @@ def latest_schema(
 
         self.LOGGER.debug("Check database is using latest schema")
         cursor = self.db_open_and_get_cursor()
-        schema_check = f"SELECT * FROM {table_name} WHERE 1=0"
+        schema_check = f"SELECT * FROM {table_name} WHERE 1=0" #nosec
```

Comment:
What if I replace the string interpolation with a safer method using `PRAGMA`  `table_info()` to retrieve schema information shown below:
`schema_check_query = f"PRAGMA table_info({table_name})"`

instead of using this `schema_check = f"SELECT * FROM {table_name} WHERE 1=0"`

Reply:
That won't solve the bandit issue, so I wouldn't waste time on it when a good table validation function can be re-used in both places and will give us the same security improvement.

Reply:
> That won't solve the bandit issue, so I wouldn't waste time on it when a good table validation function can be re-used in both places and will give us the same security improvement.

Thanks for the feedback. I will do as suggested.